submissions/Sveti-Stefan/hooli-style-popup/index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Navigation</title>
+    <link rel="stylesheet" href="styles.css" />
+</head>
+
+<body>
+    <div class="wrapper">
+        <div class="container">
+            <header class="container_header">
+                <div class="logo_header">
+                    <img src="svg/logo.svg" alt="Logo">
+                </div>
+                <nav class="menu_header_nav">

```

Comment:
Section elements typically require headings. If you should not display heading, it still must be present. Check 'visually-hidden' technique for more.

Reply:
Fixed